### ISSUE TITLE
does this work at all?

### ISSUE THREAD

**Issue by StathisKap** [twintproject/twint#1013]

**Comment by BonfaceKilz** [comment]
A better description would be welcome...

**Comment by StathisKap** [comment]
Yeah sorry, that was kinda rude. When I did twint -u elonmusk , then it would work, but as soon as i added any other parameter, then it wouldn't work. I changed some things on my PC to make it update properly, and now they work.

I do have one question though. Is there any way to get an output without the IDs? Maybe even without the dates?

**Comment by himanshudabas** [comment]
> I do have one question though. Is there any way to get an output without the IDs? Maybe even without the dates?

Yes there is, refer to the wiki, start from [here](https://github.com/twintproject/twint/wiki/Tweet-attributes#customizing).
